packages/libs/client-utils/src/built-in/deploy-contract.ts

```diff
@@ -0,0 +1,92 @@
+import type { ChainId } from '@kadena/client';
+import { Pact } from '@kadena/client';
+
+import { submitClient } from '../core';
+import type { IClientConfig } from '../core/utils/helpers';
+
+interface ITransactionBodyInput {
+  chainId: ChainId;
+  networkId: string;
+  signers: string[];
+  meta: {
+    gasLimit: number;
+    chainId: ChainId;
+    ttl: number;
+    senderAccount: string;
+  };
+  data?: { key: string; value: string };
+  keysets?: { name: string; pred: string; keys: string[] }[];
+}
```

Comment:
This already exist somewhere, right? Maybe it can extend from an existing type if you're adding properties